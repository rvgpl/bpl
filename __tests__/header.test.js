import React from 'react';
import renderer from 'react-test-renderer';
import Header from "../components/header";


  it('Header Component', () => {
    const component = renderer.create(<Header />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
