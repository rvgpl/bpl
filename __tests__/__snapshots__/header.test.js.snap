exports[`test Header Component 1`] = `null`;

exports[`test Header renders 1`] = `null`;
